frontend/src/routes/[[lang=locale]]/(voters)/Banner.svelte

```diff
@@ -0,0 +1,93 @@
+<script lang="ts">
+  import {t} from '$lib/i18n';
+  import {getContext} from 'svelte';
+  import type {CandidateContext} from '$lib/utils/candidateContext';
+  import {appType, openFeedbackModal, settings, resultsAvailable} from '$lib/stores';
+  import {Button} from '$lib/components/button';
+  import {getRoute, Route} from '$lib/utils/navigation';
+  import {LogoutButton} from '$lib/candidate/components/logoutButton';
+  import {getTopBarActionsContext} from '../topBarActions.context';
+  import {goto} from '$app/navigation';
+
+  /** Synced version so that we don't have to await for this explicitly */
+  let resultsAvailableSync = false;
+  $: $resultsAvailable.then((d) => (resultsAvailableSync = d));
+
+  const userStore = getContext<CandidateContext>('candidate')?.user;
+
+  // We are in the candidate application and the user has logged in
+  // TODO: Figure out a way to define this LogoutButton part only within the
+  // candidate route. This can be done with the new, slot-less templates
+  const showLogoutButton = $appType === 'candidate' && userStore;
+
+  const topBarActions = getTopBarActionsContext();
+</script>
+
+<!-- style:--headerIcon-color={hasVideo && screenWidth < Breakpoints.sm
+  ? 'white'
+  : 'oklch(var(--p))' -->
+<div class="vaa-basicPage-actions flex gap-0" style:--headerIcon-color="oklch(var(--p))">
+  {#if showLogoutButton}
+    <LogoutButton variant="icon" />
+  {/if}
+
+  {#if $settings.header.showFeedback && !$topBarActions.hideFeedbackButton}
+    <Button
+      on:click={$openFeedbackModal}
+      variant="icon"
+      icon="feedback"
+      text={$t('feedback.send')} />
+  {/if}
+
+  {#if $settings.header.showHelp && !$topBarActions.hideHelpButton}
+    <Button href={$getRoute(Route.Help)} variant="icon" icon="help" text={$t('help.title')} />
+  {/if}
+
+  {#if $settings.questions.showResultsLink && !$topBarActions.hideResultsButton}
```

Comment:
There might well be a better way than that I proposed, but it would be nice to be able to do these things:

1. Show feedback and help buttons on all routes if the settings say so
2. Hide either feedback or help by any layout or page
3. Show the results button on all `/questions` routes (i.e. defined by a layout) if the settings say so 
4. Show the logout button in the Candidate App only, i.e. on all `/candidate` routes

Then we could do the following:

1. `Banner` doesn't show any actions by default
2. `/+layout.svelte` sets the top bar actions: `{ help: $settings.header.showHelp ? 'show' : undefined, feedback: $settings.header.showFeedback ? 'show' : undefined }`
3. `/questions/+layout.svelte` sets: `{ results: $settings.questions.showResults ? 'show' : undefined }`
4. `/results/[entityType]/[entityId]/+page.svelte` sets: `{ help: 'hide', feedback: 'hide', return: 'show' }`
5. `/candidate/+layout.svelte` sets: `{ logout: 'show' }`